### PR TITLE
Pract.02 : Add 01-03 | Gabriela Mladenova

### PR DESCRIPTION
**Pull request by gabrielavm**
Gabriela Mladenova
SI 1 3MI0600225

### CODE REVIEW

File: Seminars/Sem.02/Pract.02/Task01.cpp [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+//SI
+//Group 1
+//FN: 3MI0600225
+
+#include <iostream>
+#include <fstream>
+
+const char FILENAME[] = "result.txt";
+
+int readFromFile(const char NAME[], int &substract)
```
Comment (khatoni) [inline Seminars/Sem.02/Pract.02/Task01.cpp]: do not use & with primitive data types

File: Seminars/Sem.02/Pract.02/Task02.cpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+size_t lengthOfFile(const char NAME[])
+{
+    std::ifstream file(NAME);
+    if (!file.is_open())
+    {
+        std::cout << "ERROR! The file could not be opened! ";
+        return 0;
+    }
+
```
Comment (khatoni) [inline Seminars/Sem.02/Pract.02/Task02.cpp:19]: Save the initial position of tellg, in order not to change the position of tellg when finding the length of the file

File: Seminars/Sem.02/Pract.02/Task03.cpp [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        std::cout << "ERROR! The file could not be opened! ";
+        return;
+    }
+
+    writeInFile << "Dear, " << title << " " << recipientName << "." << std::endl
+        << std::endl << message << std::endl
+        << std:: endl << "Sincerely, " << senderName;
+
+    writeInFile.close();
+}
```
Comment (khatoni) [inline Seminars/Sem.02/Pract.02/Task03.cpp:59]: Read the task again

**Review by khatoni — CHANGES_REQUESTED** [review]

**Comment by Justsvetoslavov** [comment]
Check uploaded solutions.
You can re-open the PR or open a new one if you have better solution or questions.

**Pull request closed without merging**